src/__tests__/index.js

```diff
@@ -205,3 +205,101 @@ test('.toBeVisible', () => {
     expect(container.querySelector('p')).toBeVisible(),
   ).toThrowError()
 })
+
+test('.toBeDisabled', () => {
+  const {queryByTestId} = render(`
+    <div>
+      <button disabled={true} data-testid="button-element">x</button>
+      <textarea disabled={true} data-testid="textarea-element"></textarea>
+      <input type="checkbox" disabled={true} data-testid="input-element" />
+
+      <fieldset disabled={true}>
+        <button data-testid="inherited-element">x</button>
+      </fieldset>
+      <div disabled={true}>
+        <button data-testid="div-child-element">x</button>
+      </div>
+
+      <fieldset disabled={true} data-testid="fieldset-element">
+        <div><div><span>
```

Comment:
Can you explain the purpose/value in these extra divs and spans?

Reply:
To make sure, that it's "deep enough". I'll correct this, it's a bit silly as I think of it right now. 
Good catch, thank you :thumbsup: